apps/wallet/src/ui/app/staking/getEarnToken.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ActiveValidator, DelegatedStake } from '@mysten/sui.js';
+
+export function getEarnToken(
```

Comment:
Is this the reward? Should at least be renamed `getEarnedTokens`, or maybe `getStakingRewards`?

Reply:
done